aws-logs-querydefinitions/template.yml

```diff
@@ -0,0 +1,23 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: AWS SAM template for the AWS::Logs::QueryDefinitions resource type
+
+Globals:
+  Function:
+    Timeout: 180  # docker start-up times can be long for SAM CLI
+    MemorySize: 256
```

Comment:
Is this the standard/recommended configuration?

Reply:
yes